.github/workflows/typescript_tests.yml

```diff
@@ -64,3 +64,38 @@ jobs:
       - run: npm test
         working-directory: ./sdks/typescript
 #        if: ${{ matrix.os != 'ubuntu-latest' }}
+
+  typescript_xlang_tests:
+    name: 'TypeScript xlang Tests'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: recursive
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Setup Beam Python
+        working-directory: ./sdks/python
+        run: |
+          pip install pip setuptools --upgrade
+          pip install -r build-requirements.txt
+          pip install 'pandas>=1.0,<1.5'
```

Comment:
Why do we end up needing a special pandas install?

Reply:
Pandas is an optional dependency for dataflow, but is used for the csv/json/parquet reads.